### PR TITLE
Add food expense policy to travel fellowship

### PR DESCRIPTION
**Pull request by gedankenstuecke**
As discussed during the board hangout on Nov 19, 2024, adding in the clarification for food expenses being eligible up to a daily cap.

### CODE REVIEW

File: Travel_fellowships.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ The OBF Event Fellowships are available for the awardees to cover conference reg
 The fellowship can be used to cover conference registration fees and direct travel costs (such as travel insurance, airfare, train, metro, bus and taxi), hotel, poster printing (if presenting a poster at the conference), and/or child-care up to a value of USD 1,000. 
 Note that rental car expenses and personal car mileage are not eligible for reimbursement. 
 Lodging for earlier than the night before the first day of the event, or beyond the night the event ends, requires justification and is subject to Board approval.
+Expenses for food during are eligible up to a daily limit of $40 (receipts are required for reimbursement).
```
Comment (nlharris) [inline Travel_fellowships.md:78]:
```suggestion
Expenses for food during the event are eligible up to a daily limit of $40 (receipts are required for reimbursement).
```

Comment (nlharris) [reply Travel_fellowships.md:78]: fix typo
otherwise looks good

**Review by nlharris — CHANGES_REQUESTED** [review]

File: Travel_fellowships.md (hunk 1 above)
Comment (gedankenstuecke) [reply Travel_fellowships.md]: Thanks @nlharris!

**Review by nlharris — APPROVED** [review]

**Review by idoerg — COMMENTED** [review]
I would fix to $40 (USD) to avoid confusing the Canadians, Australians, etc.

**Comment by peterjc** [comment]
USD 40 would match the other usage (bar one $1000 which we should fix too).

[Personally I would write USD $40 and USD $1000 as the style?]

Also, how about we move "Receipts are required for reimbursement" to the end of the paragraph - we want to be clear this applies to everything.

**Comment by gedankenstuecke** [comment]
Thanks, @idoerg & @peterjc. I've unified the currency as suggested - and also moved the bit on the receipt requirements down into the _Expenses_ section to cover both virtual and in-person events.

File: Travel_fellowships.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ This section is important for the evaluation of their application.
 
 ### Justification for funding
 
-The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD 1,000. 
+The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD $1,000.
```
Comment (hlapp) [inline Travel_fellowships.md:62]:
```suggestion
The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD 1,000. 
```
The dollar sign is ambiguous (there are a number of countries that use "dollar" as the name of their currency), and redundant with the actually very specific USD (US Dollar). (I.e., USD $1 literally reads _1 US dollar dollar_)

File: Travel_fellowships.md [hunk 3]
```diff
@@ -68,15 +68,16 @@ Please see the next section for more details.
 
 ## What does the event fellowship cover?
 
-The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD 1,000. 
+The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD $1,000.
```
Comment (hlapp) [inline Travel_fellowships.md:71]:
```suggestion
The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD 1,000. 
```
See above.

File: Travel_fellowships.md [hunk 4]
```diff
@@ -68,15 +68,16 @@ Please see the next section for more details.
 
 ## What does the event fellowship cover?
 
-The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD 1,000. 
+The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD $1,000. 
 
 ### Items covered for in-person participation
 
-The fellowship can be used to cover conference registration fees and direct travel costs (such as travel insurance, airfare, train, metro, bus and taxi), hotel, poster printing (if presenting a poster at the conference), and/or child-care up to a value of USD 1,000. 
+The fellowship can be used to cover conference registration fees and direct travel costs (such as travel insurance, airfare, train, metro, bus and taxi), hotel, poster printing (if presenting a poster at the conference), and/or child-care up to a value of USD $1,000.
```
Comment (hlapp) [inline Travel_fellowships.md:75]:
```suggestion
The fellowship can be used to cover conference registration fees and direct travel costs (such as travel insurance, airfare, train, metro, bus and taxi), hotel, poster printing (if presenting a poster at the conference), and/or child-care up to a value of USD 1,000. 
```
See above.

File: Travel_fellowships.md [hunk 5]
```diff
@@ -68,15 +68,16 @@ Please see the next section for more details.
...
 
 ### Items covered for in-person participation
 
-The fellowship can be used to cover conference registration fees and direct travel costs (such as travel insurance, airfare, train, metro, bus and taxi), hotel, poster printing (if presenting a poster at the conference), and/or child-care up to a value of USD 1,000. 
+The fellowship can be used to cover conference registration fees and direct travel costs (such as travel insurance, airfare, train, metro, bus and taxi), hotel, poster printing (if presenting a poster at the conference), and/or child-care up to a value of USD $1,000. 
 Note that rental car expenses and personal car mileage are not eligible for reimbursement. 
 Lodging for earlier than the night before the first day of the event, or beyond the night the event ends, requires justification and is subject to Board approval.
-If the applicants anticipate that they would require more than USD 1,000 to attend the event, they should include that information in the application form.
-Please note that requests made for more than $1000 may not be granted if not justified by the applicants.
+Expenses for food during the event are eligible up to a daily limit of USD $40.
```
Comment (hlapp) [inline Travel_fellowships.md:78]:
```suggestion
Expenses for food during the event are eligible up to a daily limit of USD 40.
```

File: Travel_fellowships.md [hunk 6]
```diff
@@ -68,15 +68,16 @@ Please see the next section for more details.
...
+If the applicants anticipate that they would require more than USD $1,000 to attend the event, they should include that information in the application form.
```
Comment (hlapp) [inline Travel_fellowships.md:79]:
```suggestion
If the applicants anticipate that they would require more than USD 1,000 to attend the event, they should include that information in the application form.
```

File: Travel_fellowships.md [hunk 7]
```diff
@@ -68,15 +68,16 @@ Please see the next section for more details.
...
+Please note that requests made for more than USD $1,000 may not be granted if not justified by the applicants.
```
Comment (hlapp) [inline Travel_fellowships.md:80]:
```suggestion
Please note that requests made for more than USD 1,000 may not be granted if not justified by the applicants.
```

File: Travel_fellowships.md [hunk 8]
```diff
@@ -89,11 +90,12 @@ Similarly, the cost of high-speed internet may vary across countries and may als
 Applicants are eligible to apply for the cost of a communication platform only if they are hosting a virtual event and will require a paid subscription to access the necessary premium features of the software used in their event.
 When applying for the cost of hardware, please note that this fellowship will not cover the cost of buying laptops and computers.
 However, this fellowship may cover the rental cost of the large hardware (for example, a second screen) required to attend or host a virtual event.
-The cost will be capped to a maximum of $40 (USD) each for small hardware such as headphones, webcam, speaker, and/or microphone.
+The cost will be capped to a maximum of USD $40 each for small hardware such as headphones, webcam, speaker, and/or microphone.
```
Comment (hlapp) [inline Travel_fellowships.md:93]:
```suggestion
The cost will be capped to a maximum of USD 40 each for small hardware such as headphones, webcam, speaker, and/or microphone.
```

File: Travel_fellowships.md (hunk 7 above)
Comment (hlapp) [inline Travel_fellowships.md:80]:
```suggestion
Please note that requests made for more than USD 1,000 may not be granted if not justified by the applicants.
```

**Review by hlapp — CHANGES_REQUESTED** [review]

**Comment by hlapp** [comment]
We also want to consider removing the thousands separator. Yes, it makes it easier to read, but comma is not the global standard. (According to the [Wikipedia entry](https://en.wikipedia.org/wiki/Decimal_separator) ` ` (space) is actually the international recommendation, though I'm not sure I've seen that often.)

**Comment by idoerg** [comment]
My $0.02 (USD)

1. Keep the 1,000 separator so it is easier to read. I don't think anyone will think that we are capping at $1.

2. Add a definition of USD somewhere in the beginning (e.g. "US dollars or USD")

**Review by hzahroh — APPROVED** [review]

**Comment by peterjc** [comment]
Thank you all.

**Pull request merged**